db/write_thread.h

```diff
@@ -114,6 +117,7 @@ class WriteThread {
   // Information kept for every waiting writer.
   struct Writer {
     WriteBatch* batch;
```

Comment:
Is it easy to remove `batch` in favor of `batches`?

Reply:
Not easy....